xbrowse_server/api/views.py

```diff
@@ -661,6 +662,86 @@ def add_or_edit_variant_tags(request):
     })
 
 
+@login_required
+@log_request('delete_gene_note')
+def delete_gene_note(request, note_id):
+    ret = {
+        'is_error': False,
+    }
+
+    notes = GeneNote.objects.filter(id=note_id)
```

Comment:
another minor point, but single object lookups, may be better to use this pattern instead of GeneNote.objects.filter:
```
try:
    note = GeneNote.objects.get(id=note_id)
    ...
except ObjectDoesNotExist:
    ret['is_error'] = True
```